_posts/en/2020-11-17-FWD50-in-the-open.md

```diff
@@ -0,0 +1,69 @@
+title:  "FWD50 in the open"
+ref: FWD50-2020
+lang: en
+author: "Rachel Muston and Guillaume Charest"
+date: "2020-11-20"
+excerpt: "FWD50 looked very different this year. But, there was still a lot of passion and will from participants to continuously improve government!"
+---
+<!--markdownlint-disable MD033-->
+
+FWD50 looked very different this year.
+From Nov 3-9, attendees and presenters joined the conference virtually, trading the magnificent [Pavillion Aberdeen](https://ottawa.ca/en/lansdowne#h3) for their couches and home offices.
+The virtual platform used to recreate the level of energy found in-person was [Grip](https://grip.events/), which provided agenda information and the ability for the users to build up their schedule and setup impromptu networking opportunities to maximize their time during the four-day conference.
+
+Each session had a [Vimeo](https://vimeo.com) channel where folks could watch the video with closed captioning and workshops used dedicated [Zoom](https://zoom.us/) rooms to provide interactive features to the participants.
+
+On the side of each session display, a text chat feature allowed participants to share information with each other and ask questions to the presenters, which opened up really interesting discussions and further improved networking opportunities.
+
+During each session, there was also an opportunity for the users to tag a presenter to ask to network with them later.
+
+Outside the main platform, there were virtual break out rooms where the participants could move their avatar to bump into friends and new acquaintances alike and have the sort of spontaneous conversations that happen at in-person conferences.
+The services used to create those virtual rooms were [Kumospace](https://www.kumospace.com/) and [Wonder](https://www.wonder.me/).
+
+The FWD50 team were very upfront at the beginning to say that there will likely be some hiccups along the way, and while there was one or two minor ones, the conference went very smoothly.
+
+And the good news is that while a paid ticket was required to attend, a lot of content was and still is freely available.
+
+Some of the presenters have made their presentations or details of their presentation open:
+
+- [Pia Andrews](https://twitter.com/piacandrews?lang=en) presented on [The Crisis Paradox For Leadership: Balancing Urgency and Importance](https://t.co/RELqF74GgK?amp=1)
+- [Honey Ducanay's](https://twitter.com/honeygolightly?lang=en) part of the Delivering of the Speed of Need workshop entitled [Enabling conditions, not just heroics](https://t.co/a7u53IwYqs) is available on her blog
+- Also from the 'Delivering at the Speed of Need workshop, panelists [Katherine Benjamin](https://twitter.com/mskatiebenjamin), Honey Ducanay and [Katy Lalonde](https://twitter.com/katylalonde) made their [cheat sheet on delivering at the speed of need](https://docs.google.com/presentation/d/1gL57LdKFFZOm5ngL1HqHubzzG80aLewDcJY8hTDqsaQ/edit#slide=id.p) openly available
```

Comment:
there's an appostrophe too much here. Change:
Also from the 'Delivering at the Speed of Need workshop

to

Also from the 'Delivering at the Speed of Need' workshop